src/main/java/spoon/support/reflect/declaration/CtTypeImpl.java

```diff
@@ -345,6 +345,11 @@ public boolean isAnonymous() {
 	}
 
 	@Override
+	public boolean isLocal() {
+		return getReference().isLocal();
```

Comment:
here we can do better than the naming convention: `getParent(CtMethod)!=null && getParent()!=CtNewClass` 

correct?